src/app/page.tsx

```
@@ -1,248 +1,23 @@
-              <li>
-                2. 위의 버튼들을 클릭하여 각 API 엔드포인트를 테스트하세요.
-              </li>
-              <li>3. 응답 결과가 JSON 형태로 표시됩니다.</li>
-            </ol>
-          </div>
-        </div>
-      </div>
+    <div className='h-screen w-full'>
+      <main
+        id='main-content'
+        role='main'
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<main>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)